oak/proto/manager.proto

```diff
@@ -72,34 +72,9 @@ message StorageProxyConfiguration {
   string address = 1;
 }
 
-message CreateApplicationRequest {
-  ApplicationConfiguration application_configuration = 1;
-}
-
-message CreateApplicationResponse {
-  // The unique ID of the newly created Application.
-  string application_id = 1;
-
-  // The port on which the Application is listening over gRPC.
-  int32 grpc_port = 2;
-}
-
-message TerminateApplicationRequest {
-  string application_id = 1;
-}
-
-message TerminateApplicationResponse {
-}
-
-// Untrusted service in charge of creating Oak Applications on demand.
-service Manager {
-  // Request the creation of a new Oak Application with the specified configuration.
-  //
-  // After the Oak Node is created, the client should connect to the returned
-  // endpoint via gRPC and perform a direct attestation against the Node itself,
-  // to verify that its configuration corresponds to what the client expects.
-  rpc CreateApplication(CreateApplicationRequest) returns (CreateApplicationResponse);
-
-  // Request that an Oak Application terminate.
-  rpc TerminateApplication(TerminateApplicationRequest) returns (TerminateApplicationResponse);
+// ApplicationCreationStatus describes the configuration of a successfully
+// loaded Oak application.
+message ApplicationCreationStatus {
+  // The port on which the application is listening over gRPC.
+  int32 grpc_port = 1;
```

Comment:
Since we now expect exactly one instance of a gRPC server to be running for each invocation, I think it's reasonable to assume that a specific port will be passed in -- also it is necessary for it to work with docker, and I don't think we have any use case in which we parse the text output of the loader to figure out what the actual port is. I think we should validate that the requested port in the app configuration is valid (i.e. strictly > 0) and not even bother with the logic for dynamic port assignment, and also remove this field (and perhaps the entire message, if you think we won't need it for anything else anytime soon). WDYT?